### PR TITLE
Update FedRAMP navigation templates

### PR DESCRIPTION
**Pull request by shawndwells**
Updates the FedRAMP Navigation menu:

image

### CODE REVIEW

File: assets/src/app/AppLayout/Navigation.tsx [hunk 1]
```diff
@@ -67,11 +67,8 @@ interface NavigationState {
 
 const staticNavigation:(MyRouterItem)[] = [
     {label: 'Getting Started', to: '/ato/getting_started'},
-    {label: 'Documents', routes: [
-        {label: 'Overview', to: '/ato/documents'},
-        {label: 'Vulnerability Management', to: '/ato/documents/vulnerability-management-plan'},
-        {label: 'Security Awareness', to: '/ato/documents/security-awareness-and-training-plan'},
-        {label: 'FedRAMP Templates', to: '/ato/documents/fedramp-templates'},
+    {label: 'FedRAMP Resources', routes: [
+        {label: 'SSP Templates', to: '/ato/documents/fedramp-templates'},
```
Comment (shawndwells) [inline assets/src/app/AppLayout/Navigation.tsx]: Removing the old ``Documents`` section was intentional. Docs haven't been maintained in year(s) and were never completed to begin with.

File: assets/src/app/AppLayout/Navigation.tsx [hunk 2]
```diff
@@ -93,7 +90,6 @@ const staticNavigation:(MyRouterItem)[] = [
         {label: 'Systems and Communications Protection', productTo: '/ato/products/select/NIST-800-53#SC-'},
         {label: 'System and Information Integrity', productTo: '/ato/products/select/NIST-800-53#SI-'},
     ]},
-    {label: 'FedRAMP', productTo: '/ato/products/select/FedRAMP'},
```
Comment (shawndwells) [inline assets/src/app/AppLayout/Navigation.tsx]: Removing for now, until product-level navigation is sorted.

File: assets/src/app/ato/Documents/Fedramp.tsx [hunk 3]
```diff
@@ -12,7 +12,7 @@ import * as Api from '@app/lib/api'
 interface FedRAMPsState {
     rows: IRow[];
 }
-class ATOFedRAMPs extends React.Component<any, FedRAMPsState> {
+class ComponentFedRAMPTemplates extends React.Component<any, FedRAMPsState> {
```
Comment (shawndwells) [inline assets/src/app/ato/Documents/Fedramp.tsx]: Seem to recall PatternFly/react using this naming scheme... but cannot cite sources. Am I going crazy? Can easily revert this change.

File: assets/src/app/ato/Documents/Fedramp.tsx [hunk 4]
```diff
@@ -28,6 +28,11 @@ class ATOFedRAMPs extends React.Component<any, FedRAMPsState> {
         }).map(function(p, idx) {
             return [
                 (<Text component="p" key={idx}>{p.name}</Text>),
+                /* (<TextContent key={idx+'t'}>
+                <center>
+                    <Text component="a" href={'/api/v1/components/' + p.key + '/fedramp/Tailored'}><FileWordIcon/> .docx</Text>
```
Comment (isimluk) [inline assets/src/app/ato/Documents/Fedramp.tsx]: `/fedramp/Tailored` is not provided by the backend.

What is the tailored supposed to do?

File: assets/src/app/ato/Documents/Fedramp.tsx (hunk 3 above)
Comment (isimluk) [reply assets/src/app/ato/Documents/Fedramp.tsx]: I didn't noticed many components starting with component keyword.

Nevertheless, no worries, the name can remain what it is.

File: assets/src/app/ato/Products/Static.tsx [hunk 5]
```diff
@@ -45,7 +44,6 @@ const ProductInfo: {[Identifier: string]: ProductTemplate } = {
         }
     },
     insights: {
-        disableFedrampDownload: true,
```
Comment (isimluk) [inline assets/src/app/ato/Products/Static.tsx]: I thought we wanted to not yet show the fedramp for insights?

**Comment by isimluk** [comment]
Thanks! Will merge and amend minor stuff in separate commits.
